chapter2/README.md

```diff
@@ -15,7 +15,8 @@
 
 ## Overview
 
-This sample demonstrates a cross-platform application suite involving an Angular SPA (*TodoListSPA*) calling an ASP.NET Core web API (*TodoListAPI*) secured with **Azure Active Directory**. It also implements authorization for **role-based access control** using Azure AD **Security Groups**. In the sample, a dashboard component allows signed-in users to see the tasks assigned to them or other users based on their memberships to one of the two security groups, **GroupAdmin** and **GroupMember**.
+This sample demonstrates a cross-platform application suite involving an Angular SPA (*TodoListSPA*) calling an ASP.NET Core web API (*TodoListAPI*) secured with **Azure Active Directory**. It also implements authorization for **role-based access control** using Azure AD **Security Groups**. 
```

Comment:
>This sample demonstrates [](start = 0, length = 24)

Start with "In [chapter 1](link) we learnt how to use approles and then in the second chapter.."